generate_ssl_certs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir certs
+openssl req -x509 -newkey rsa:4096 -keyout certs/key.pem -out certs/cert.pem -nodes -days 900
```

Comment:
Suggestion: Instead of using self-signed certificates, we could try something like [Let's Encrypt](https://letsencrypt.org/), or if we're going to use self-signed certificates maybe we don't need a script for that, we could just add these lines to our documentation.